examples/Person.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Excel.IO.examples
```

Comment:
Could we capitalise Examples:

```suggestion
namespace Excel.IO.Examples
```